### PR TITLE
Chap1 testing

### PR DESCRIPTION
**Pull request by sritej-reddy**

### CODE REVIEW

File: C-Unit1/src/E01_Functions.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+// sum of two 100 digit positive numbers
+//
+void sumOf100DigitNumbers(int num1[100], int num2[100], int sum[101]) {
+	int i,carry=0,temp;
+	for (i = 99; i > -1; i--)
+	{
+		temp = num1[i] + num2[i] + carry;
+		sum[i + 1] = temp % 10;
+		carry = temp / 10;
+	}
```
Comment (krushiraj) [inline C-Unit1/src/E01_Functions.cpp:62]: Did the tests pass for this?

**Review by krushiraj — CHANGES_REQUESTED** [review]
Just test all the functions at once. Then just post a screenshot of it here as a comment. Everything else is good to go and ready to be merged.

File: C-Unit1/src/E01_Functions.cpp (hunk 1 above)
Comment (sritej-reddy) [reply C-Unit1/src/E01_Functions.cpp:62]: Yes all the test cases passed

Comment (sritej-reddy) [reply C-Unit1/src/E01_Functions.cpp:62]: I'll send the screenshot in the evening anna

Comment (sritej-reddy) [reply C-Unit1/src/E01_Functions.cpp:62]: image

Comment (krushiraj) [reply C-Unit1/src/E01_Functions.cpp:62]: Okay good to go. We'll merge this.

**Pull request merged**